common/services/whats-new-content.js

```diff
@@ -36,7 +36,7 @@ const service = {
     space: CONTENTFUL_SPACE_ID,
     accessToken: CONTENTFUL_ACCESS_TOKEN,
   }),
-  fetchEntries: async () => {
+  fetchWhatsNewContent: async () => {
```

Comment:
I reckon we could call this "fetch", since we're already in the what's new service, it can only be what's new content.

Reply:
Good shout